_posts/en/posts/2016-06-24-segwit-next-steps.md

```diff
@@ -0,0 +1,166 @@
+
+The high bandwidth mode is being used as the basis for further development into optimizations for peer-to-peer block relay.  It's especially intended for use by miners who need to receive blocks as fast as possible (especially if other non-peer-to-peer block relay methods fail), but users with extra bandwidth can also enable this mode to help relay blocks to miners faster and also help discourage denial-of-service attacks by making it less evident which high-bandwidth nodes are being run by miners and which are being run by ordinary peers.
+
+## Deployment plan
+
+The following plan describes how segwit is expected to be deployed.
+
+**Merge to master (without mainnet activation code):** after Bitcoin Core developers "ACK" (approve) the final segwit pull request, it will be merged into the Bitcoin Core master Git repository branch.  The code that is being merged will include everything in segwit except for the activation code.  This will make it easy for developers to test other features on top of segwit, such as compact blocks.
+
+**Backport to 0.12 branch:** the unactivated code will be backported to the 0.12 maintenance branch and the backport will receive its own testing.
+
+**Choosing the BIP9 parameters:** [BIP9][] is a soft fork deployment mechanism that allows miners to signal their readiness to enforce new consensus rules.  Each soft fork made with BIP9 chooses when miners can begin signaling for the soft fork, when the soft fork is considered unsuccessful if not enough miners have signaled for it, and which bit in the block header version field will be used by miners to signal their readiness.  These parameters will be selected at this time and implemented along with the code to activate segwit on both the master and 0.12 branches.
+
+**Release candidate phase:** after all developer testing is successfully concluded, a release candidate (probably named 0.12.2RC1) will be publicly provided to anyone willing to test the code.  Miners, merchants, and wallet vendors are especially encouraged to test.  If any problems are found, they will be fixed and a new release candidate will be issued.  This will be repeated as necessary until a release candidate is found with no known problems.
+
+**Binary release:** the final release candidate will have its version changed to the final release version (expected to be 0.12.2) and will be released for all users to download and begin running at their leisure (segwit is a soft fork, so upgrading is only required if they plan to use segwit features; however upgrade is appreciated).
```

Comment:
"upgrade is appreciated" sounds a bit strange to me - appreciated by whom - maybe recommended?